README.md

```diff
@@ -2,73 +2,55 @@
 | :---- | :------ | :---- |
 | [![Build Status](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_apis/build/status/HostedWindowsContainer-CI)](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_build/latest?definitionId=4) | [![Build Status](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_apis/build/status/HostedmacOS-CI)](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_build/latest?definitionId=6) | [![Build Status](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_apis/build/status/HostedUbuntu1604-CI)](https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_build/latest?definitionId=5)
 
-Stratis Bitcoin
+Blockcore
 ===============
 
-https://stratisplatform.com
+https://blockcore.net
 
 Bitcoin Implementation in C#
 ----------------------------
 
-Stratis is an implementation of the Bitcoin protocol in C# on the [.NET Core](https://dotnet.github.io/) platform.  
-The node can run on the Bitcoin and Stratis networks.  
-Stratis Bitcoin is based on the [NBitcoin](https://github.com/MetacoSA/NBitcoin) project.  
+**What is Blockcore?**
 
-For Proof of Stake support on the Stratis token the node is using [NStratis](https://github.com/stratisproject/NStratis) which is a POS implementation of NBitcoin.  
+- Blockcore is a [.NET Core](https://dotnet.github.io/) platform to build decentralized networks based on the Bitcoin protocol
+- Blockcore is a platform to build Layer 1 consensus networks based on the Bitcoin protocol, built on the [.NET Core](https://dotnet.github.io/) framework and written entirely in C#. 
+- Blockcore aims to maintain an alternative C# Bitcoin implementation, based on the [NBitcoin](https://github.com/MetacoSA/NBitcoin) & [Stratis](https://github.com/stratisproject/StratisBitcoinFullNode) projects.
+- Blockcore isn't a coin.
+- Blockcore isn't a for profit business.
 
-[.NET Core](https://dotnet.github.io/) is an open source cross platform framework and enables the development of applications and services on Windows, macOS and Linux.  
+**Why Blockcore?**
 
-Join our community on [discord](https://discord.gg/9tDyfZs).  
+- We see a need within the crypto ecosystem for development of the C# full node technology.
+- Stratis has provided an excellent starting point but their focus is enterprise and businesses and we feel strongly that there is also value focusing on open and public blockchains.
```

Comment:
I don't agree with this statement, I think blockcore should very much focus on enterprise and business use. This could instead be formulated to be something around that "blockcore is suppose to be a blockchain-platform that can enable anyone to easily develop and launch their own blockchains, both public and private"

Reply:
I will remove the first about Blockcore statement as it is duplicated in the second.

I wanted to be clear that we aren't a business & we aren't competing with the Stratis business model.